### PR TITLE
Update home page for Fall 2016 (+ remove outdated routes)

### PR DESCRIPTION
**Pull request by warrenshen**

### CODE REVIEW

File: app/controllers/pages_controller.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@ def home
     @check_in = CheckIn.new
   end
 
-  def why_this_class
+  def why
```
Comment (charlesx2013) [inline app/controllers/pages_controller.rb]: but why not y

**Comment by charlesx2013** [comment]
85% sure this is failing because you didnt update the tests.

**Comment by charlesx2013** [comment]
Also, update the time and schedule.

File: config/routes.rb [hunk 2]
```diff
@@ -1,57 +1,62 @@
...
+  get "apply", to: "student_applications#new", as: "apply"
+  post "apply", to: "student_applications#create", as: "student_applications"
+  get "office-hours", to: "pages#office_hours"
+  get "why", to: "pages#why"
 
-  resources :lectures, only: [:show, :index]
-  resources :student_applications, only: [:new, :create]
-
-  devise_for :users, path: '',
+  devise_for :users, path: "",
```
Comment (charlesx2013) [inline config/routes.rb]: single quotes

Comment (warrenshen) [inline config/routes.rb]: What isn't everything else in double quotes?

**Pull request merged**